app/gui/view/graph-editor/src/component/node.rs

```diff
@@ -332,7 +332,10 @@ ensogl::define_endpoints_2! {
         visualization_visible    (bool),
         visualization_path       (Option<visualization::Path>),
         expression_label_visible (bool),
-        bounding_box             (BoundingBox)
+        bounding_box             (BoundingBox),
+        /// The [`display::object::Model::position`] of the Node. The position change events are
+        /// emitted after corresponding [`bounding_box`] change events.
+        position                 (Vector2),
```

Comment:
Given that the bounding_box has no description, I have no idea when the position changes will be re-emitted. I think it should be stated clearly: the position (and bbox) change will be emitted only when Display Object hierarchy is updated (with reference to update method).